.gitignore

```diff
@@ -4,6 +4,7 @@
 /mod/*
 composer.lock
 composer.phar
+cache
```

Comment:
Do these need to start with `/`? If we have a view name containing "cache" we don't want to ignore it right?